Dockerfile

```
@@ -1,31 +1,37 @@
 FROM ubuntu:20.04
+LABEL version="0.1.0"
+LABEL authors="Marco Favorito <favorito@diag.uniroma1.it>, Francesco Fuggitti <fuggitti@diag.uniroma1.it>"
```

I cannot be considered an author _yet_. What about this one?
```suggestion
LABEL authors="Marco Favorito <favorito@diag.uniroma1.it>"
LABEL contributors="Francesco Fuggitti <fuggitti@diag.uniroma1.it>"
```